### PR TITLE
Admin in db

### PR DESCRIPTION
**Pull request by mcarans**
This has all the changes in the Tables in SQLAlchemy ORM format PR https://github.com/OCHA-DAP/hapi-pipelines/pull/8 as well as code to populate admin tables

This is a very inefficient implementation at present

### CODE REVIEW

File: src/hapi/pipelines/app/pipelines.py
Comment (turnerm) [inline src/hapi/pipelines/app/pipelines.py:32]: I know this was done in a different PR, but should we limit locations (and subsequently admin regions) just to HRPs for now?

**Review by turnerm — APPROVED** [review]
Looks good, just some small comments

**Comment by turnerm** [comment]
Actually something I realized: we need to include the empty connector references, i.e. for each admin1, a row in admin2 that has an undefined pcode that references the admin1. And same for admin1 to location.

File: src/hapi/pipelines/utilities/admins.py
Comment (mcarans) [reply src/hapi/pipelines/utilities/admins.py:35]: Yes good idea

File: src/hapi/pipelines/app/pipelines.py
Comment (mcarans) [reply src/hapi/pipelines/app/pipelines.py:34]: The AdminLevel class does the fuzzy matching etc., but I need to have it be set up from the same source as the Admins class (ie. the dataset). This requires changes to HDX Python Country.

File: src/hapi/pipelines/utilities/admins.py
Comment (mcarans) [reply src/hapi/pipelines/utilities/admins.py:41]: I am not sure what you have in mind. I'm happy for you to make a PR to structure it better

File: src/hapi/pipelines/utilities/admins.py
Comment (mcarans) [reply src/hapi/pipelines/utilities/admins.py:58]: Yes good idea

File: src/hapi/pipelines/utilities/admins.py
Comment (mcarans) [reply src/hapi/pipelines/utilities/admins.py:63]: Yes thanks, will remove

File: src/hapi/pipelines/utilities/admins.py
Comment (mcarans) [reply src/hapi/pipelines/utilities/admins.py:65]: Good spot.

File: src/hapi/pipelines/utilities/admins.py
Comment (mcarans) [reply src/hapi/pipelines/utilities/admins.py:66]: I would think yes, but I defer to @b-j-mills on this

File: src/hapi/pipelines/app/pipelines.py
Comment (mcarans) [reply src/hapi/pipelines/app/pipelines.py:32]: I can make that change in a new PR

Comment (mcarans) [reply src/hapi/pipelines/app/pipelines.py:32]: Actually I'll do it in this one

**Comment by mcarans** [comment]
> Actually something I realized: we need to include the empty connector references, i.e. for each admin1, a row in admin2 that has an undefined pcode that references the admin1. And same for admin1 to location.
> 
> I can take care of it in the population or a separate PR, let me know.

Yes please go ahead. I think we're waiting for @b-j-mills feedback on what to use for XXX in KEN-ABB-XXX.

File: src/hapi/pipelines/app/pipelines.py
Comment (mcarans) [reply src/hapi/pipelines/app/pipelines.py:34]: I'll look at this in a separate PR

**Pull request merged**
